ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -21,6 +21,12 @@ parameters:
 - name: BASE_DOMAIN
   value: origin-ci-int-aws.dev.rhcloud.com
   required: true
+- name: IMAGE_EGRESS_HTTP_PROXY
+  required: true
+  value: http://registry.svc.ci.openshift.org/origin/4.2:egress-http-proxy
```

Comment:
I'm pretty sure you don't want the `http://` prefix.  And this should be pretty stable; will we need to override it on a per-job basis?  Maybe hard-code it in the template for now, and follow-up work can add ways for specific jobs to customize the proxy if we end up needing per-job customization?